### PR TITLE
feat(TESB-27670): build migration

### PR DESCRIPTION
**Pull request by jgreffe**

### CODE REVIEW

File: examples/camel/pom.xml [hunk 1]
```diff
@@ -20,6 +20,8 @@
     </parent>
 
     <properties>
+        <!-- as current version is a release and not snapshot, we avoid deployment -->
+        <maven.deploy.skip>true</maven.deploy.skip>
```
Comment (xldai) [inline examples/camel/pom.xml:24]: "7.4.1-SNAPSHOT" is a snapshot. maybe I misunderstand here

Comment (jgreffe) [reply examples/camel/pom.xml:24]: No, version is 1.0 :/

Comment (xldai) [reply examples/camel/pom.xml:24]: Aha, I see.
We didn't notice it has "1.0" as the version. it's not good we publish the "1.0" artifacts for every our GA release.

Comment (andreasmattes) [reply examples/camel/pom.xml:24]: As far as I understand it, these examples are meant to be tried out locally and not for further deployment and installation

**Pull request merged**